admin/package-files/systemd/flocker-agent.service

```diff
@@ -1,11 +1,11 @@
 [Unit]
-Description=Flocker ZFS Agent
+Description=Flocker Dataset Agent
 AssertFileNotEmpty=/etc/flocker/agent.yml
 
 [Service]
-ExecStart=/usr/sbin/flocker-zfs-agent
+ExecStart=/usr/sbin/flocker-dataset-agent
 
-# We can't have a private mount namespace, since we need to see ZFS created mounts.
+# We can't have a private mount namespace, since we need to create mounts visible to Docker.
```

Comment:
I cheated and stole it from https://github.com/ClusterHQ/flocker/pull/1407/files to avoid merge conflicts.

> It's a shame we don't call it `flocker-dataset-agent.service`...for consistency with the process it launches.

Now we're actually calling it, it seems like the perfect time to change this. I'll have a look to see how hard that would be.